README.md

```diff
@@ -8,6 +8,7 @@ UDS Package Kit is a tool designed to assist in developing, maintaining, and pub
 
 - Automated release and tag creation in GitLab and GitHub
 - Customizable release configuration file
+- Comparing grype scans using the cyclonedx-json format
```

Comment:
Curious why we chose CycloneDX rather than the Syft?  Zarf uses Syft for its interchange and Grype supports it as well.

Reply:
I realize that changing this is effectively a rewrite of the entire PR but I strongly encourage you to do so for consistency with the rest of our SBOM tooling.

Reply:
> Curious why we chose CycloneDX rather than the Syft?

Thanks to a good offline chat with @corang - for some reason Grype doesn't output Syft (which is wild because they created it) and the native JSON format does not fit this problem as well as Cyclone.

@corang plz plz make ^^ clear in the README that we prefer the Syft SBOM format but the tooling doesn't enable that.  This does make some sense given Cyclone's more native OpenVEX support but caught me by surprise.

Reply:
fixed in next commit